### DISCUSSION TITLE
Power outage question

### DISCUSSION THREAD

**Discussion by gabrielke** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#121] [Q&A]
As I understand this project it implies the spa is basically turned on (so power is green).
This doesn't change the behavior after a power outage (whole thing remains turned off as factory default, right?)

Or it can "press" the power button too somehow?

**Comment by visualapproach** [comment]
6-wire version lets you choose what happens after power up. Default (off) or last state.
4-wire version should start in a hands-off state. Meaning we don't control the pump until we hit "take control"
The factory electronics is never really OFF until you unplug the pump. It needs to be able to protect against freezing.
